mkdocs.yml

```diff
@@ -134,33 +134,35 @@ nav:
       - Getting Started: quick-start/getting-started.md
       - Installing Configurator: quick-start/installing-configurator.md
       - Firmware Options: quick-start/firmware-options.md
-      - TX Modules:
+      - Transmitters:
           - Preparing your Radio: quick-start/transmitters/tx-prep.md
-          - Flashing TX Modules:
+          - Flashing Transmitters:
             - Axis Thor: quick-start/transmitters/axis-thor.md
             - BetaFPV 2.4GHz: quick-start/transmitters/betafpv2400.md
             - BetaFPV 900MHz: quick-start/transmitters/betafpv900.md
+            - DIY 900TX: quick-start/transmitters/diy900.md
```

Comment:
The reason the DIY and the Internal modules were lower down the list is that they can be considered as "special" modules.

I was hoping to group them together without creating a new folder in the menu. 
The first items in this list were all external modules, with almost the same Flashing/Updating steps.
The internal modules from different manufacturers have varying Flashing/Updating methodology, and their own intricacies.

The DIY modules, are that, DIY, and often only undertaken by those willing to dirty their hands. And if they are willing to dirty their hands, they are usually very resourceful and usually don't need such guides. 
And most users have also migrated into using off-the-shelf tx modules.